src/components/Confirmation.vue

```diff
@@ -30,25 +30,26 @@ export default {
 
 <style scoped lang="scss">
 .confirmation {
-  position: absolute;
-  width: 410px;
+  position: fixed;
+  z-index: 10;
   height: 41px;
   left: 295px;
   top: 16px;
   background: #ffffff;
   border-radius: 6px;
+  border: 1px solid #2b6693;
   display: flex;
   flex-direction: row;
 
   &-left {
     // position: absolute;
-    width: 43px;
-    height: 41px;
+    width: 41px;
+    height: 39px;
```

Comment:
These numbers look arbitrary. Usually these UI constants (width, margin, padding, etc) should be divisible by 8 (e.g. Android design guideline requires padding to be 24dp). Having these weird values looks like some temporary hacks.

Reply:
@tcho6319

Reply:
I got the original numbers from the designs but I updated most of the UI constants now to be divisible by 8.